pkg/cloudprovider/aws/aws.go

```diff
@@ -156,7 +156,12 @@ func (n *NodeGroup) IncreaseSize(delta int64) error {
 		return fmt.Errorf("size increase must be positive")
 	}
 
-	if n.TargetSize()+delta > n.MaxSize() {
+	// Eligible for scaling up
```

Comment:
Are you able to remove the ceiling clamping logic out of the cloud provider now that we have it in the controller?